### DISCUSSION TITLE
When using motion-safe and motion-reduce it does not output keyframes CSS

### DISCUSSION THREAD

**Discussion by zizther** [tailwindlabs/tailwindcss#7505] [Help]
**What version of Tailwind CSS are you using?**

v3.0.23

**What build tool (or framework if it abstracts the build tool) are you using?**

gulp 4.0.2, gulp-postcss 9.01, postcss 8.4.6, webpack 5.69

**What version of Node.js are you using?**

v16.13.1

**What browser are you using?**

Chrome and Brave

**What operating system are you using?**

macOS v12.1

**Describe your issue**

When using `motion-safe` or `motion-reduce` with supported animations (such as pulse) or custom animations with custom keyframes the CSS output has empty keyframes.
I have tried via PostCSS via gulp-postcss and the CLI, both return the same issue.

I am using these PostCSS plugins `postcss-import`, `autoprefixer` and `tailwindcss/nesting`

```
@media (prefers-reduced-motion: no-preference) {
    @-webkit-keyframes noise {}
    @keyframes noise {}
    .noise {
        -webkit-animation: noise 1s steps(4) infinite;
        animation: noise 1s steps(4) infinite;
    }
}
```

I have tried this on Play and it works fine https://play.tailwindcss.com/FPrYZaVZIA
I also tried with Tailwind CSS v3.0.18, but still had the same output via the CLI and PostCSS via gulp-postcss. Not sure what may be causing this with my local dev env.

Is there anything else I can try to resolve this issue?

My tailwind.config.js file looks like
```
module.exports = {
    content: [
        './templates/**/*.{html,twig,html.twig}',
        './src/js/**/*.{js,jsx,ts,tsx}',
    ],
    darkMode: 'class',
    theme: {
        extend: {
            animation: {
                noise: 'noise 1s steps(4) infinite',
            },
            keyframes: {
                noise: {
                    '0%': { transform: 'translate3d(0, 0, 0)' },
                    '10%': { transform: 'translate3d(-5%, -5%, 0)' },
                    '20%': { transform: 'translate3d(-10%, 5%, 0)' },
                    '30%': { transform: 'translate3d(5%, -10%, 0)' },
                    '40%': { transform: 'translate3d(-5%, 15%, 0)' },
                    '50%': { transform: 'translate3d(-10%, 5%, 0)' },
                    '60%': { transform: 'translate3d(15%, 15%, 0)' },
                    '70%': { transform: 'translate3d(15%, 10%, 0)' },
                    '80%': { transform: 'translate3d(-15%, -15%, 0)' },
                    '90%': { transform: 'translate3d(10%, 5%, 0)' },
                    'to': { transform: 'translate3d(5%, 5%, 0)' }
                },
            },
        },
    },
    plugins: [],
}
```

**Comment by adamwathan** [comment]
Hey! Can you share a project that _does_ reproduce the issue?

**Comment by davidurco** [comment]
Can we please fix this? Workaround for now is to ignore motion-safe and thus degrade user experience.

**Reply by wongjn** [reply]
If you'd like to expedite a fix, consider providing a project that reproduces the undesired behavior in a new issue.
